blog/_posts/2017-02-27-datapackage-pipelines.md

```diff
@@ -0,0 +1,310 @@
+---
+author: Adam Kariv
+title: Data Package Pipelines
+username: akariv
+---
+
+*[datapackage-pipelines][dpp] is the newest part of the Frictionless
+ Data toolchain.  It is a framework for defining data processing steps
+ declaratively to generate self-describing Data Packages.*
+
+[OpenSpending][os] is an open database for
```

Comment:
This doesn't flow very well from "DPP is" to "OpenSpending is". Perhaps a smoother transition would be something like "We were searching for an ETL solution for OpenSpending" or something similar which doesn't put the focus first on OpenSpending.